exporters/otlp/otlpmetric/otlpmetricgrpc/client_test.go

```diff
@@ -93,7 +93,7 @@ func newGRPCExporter(t *testing.T, ctx context.Context, endpoint string, additio
 }
 
 func newExporterEndToEndTest(t *testing.T, additionalOpts []otlpmetricgrpc.Option) {
-	mc := runMockCollectorAtEndpoint(t, "localhost:56561")
+	mc := runMockCollectorAtEndpoint(t, "localhost:0")
```

Comment:
This is equivalent to [`runMockCollector(t)`](https://github.com/open-telemetry/opentelemetry-go/blob/1eb317baec38c2204247c28a8a8bcd13cdfa5e8f/exporters/otlp/otlpmetric/otlpmetricgrpc/mock_collector_test.go#L142-L145)

```suggestion
	mc := runMockCollector(t)
```